docs/index.md

```diff
@@ -22,12 +22,9 @@ provider "lightstep" {
 <!-- schema generated by tfplugindocs -->
 ## Schema
 
-### Required
-
-- `organization` (String) The name of the Lightstep organization
-
 ### Optional
 
 - `api_key` (String) The API Key for a Lightstep organization.
 - `api_key_env_var` (String) Environment variable for Lightstep api key.
 - `environment` (String) The name of the Lightstep environment, must be one of: staging, meta, public.
+- `organization` (String) The name of the Lightstep organization
```

Comment:
looks like this got moved down here to reflect the fact that this required arg has a default (falls back to the environment key) https://github.com/lightstep/terraform-provider-lightstep/blob/c50ea86e0ea1f9019758ed6d732dd75e90b9a8d9/lightstep/provider.go#L24

i dont really have an opinion, the plugin did this. im happy to move it back